redfish/resource_redfish_power_test.go

```diff
@@ -0,0 +1,127 @@
+package redfish
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"testing"
+)
+
+// redfish.Power represents a concrete Go type that represents an API resource
+func TestAccRedfishPower_basic(t *testing.T) {
+
+	resource.Test(t, resource.TestCase{
+		Providers:    testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccRedfishResourcePowerConfig(
+														    "root",
+					                                		"password",
+					                                		"192.168.1.63",
+														 "On",
+														  120,
+												  				10),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("redfish_power.system_power", "power_state", "On"),
+				),
+			},
+			{
+				Config: testAccRedfishResourcePowerConfig(
```

Comment:
By using what I said in the other comment, we would avoid hardcoding values around (credentials related) and we could only pass the ones regarding resources.